api/camunda/camunda-openapi.yaml

```
@@ -3927,6 +3991,64 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/UserResponse"
+    RoleCreateRequest:
+      type: "object"
+      properties:
+        name:
+          type: "string"
+          description: The display name of the new role
+    RoleCreateResponse:
+      type: "object"
+      properties:
+        roleKey:
+          description: The key of the created role
```

```suggestion
          description: The key of the created role.
```

I'll port this upstream.